src/main/java/com/teamabnormals/abnormals_core/core/api/IAgeableEntity.java

```diff
@@ -1,15 +1,30 @@
 package com.teamabnormals.abnormals_core.core.api;
 
-/***
- * @author tessdotcpp
- * Use to make an entity that doesn't extend AgeableEntity compatible with Quark's potato poisoning.
+import net.minecraft.entity.LivingEntity;
+
+/**
+ * Use to make a living entity that doesn't extend AgeableEntity compatible with Quark's potato poisoning and
+ * Savage & Ravage's Growth & Youth Potions.
+ * <p>Classes implementing this must extend {@link LivingEntity}.</p>
+ *
+ * @author abigailfails
  */
 public interface IAgeableEntity {
     /**
-     * Sets the growing age of the entity. With a negative value it's considered a child; use this method to check for
-     * an age of 0 or greater and trigger the necessary changes.
+     * If the entity can grow to a higher stage, resets any progress made towards it (e.g. age timer).
+     */
+    void resetGrowthProgress();
+
+    /**
+     * @param growingUp True if this should check for a higher growth stage or false for a lower one.
+     * @return If the entity can grow or regress into another stage depending on growingUp.
      */
-    void setGrowingAge(int age);
+    boolean canAge(boolean growingUp);
 
-    int getGrowingAge();
+    /**
+     * Attempts to change the entity's growth stage depending on isGrowingUp.
```

Comment:
Still `isGrowingUp` here.
```suggestion
     * Attempts to change the entity's growth stage depending on growingUp.
```

Reply:
Oh right, I forgot to change that.